utils/node_local_test.py

```diff
@@ -2196,7 +2169,7 @@ def test_complex_unlink(self):
         # Create both files in the dfuse instance we're checking.  These files are created in
         # binary mode with buffering off so the writes are sent direct to the kernel.
         for index in range(2):
-            fd = open(os.path.join(self.dfuse.dir, 'file.{}'.format(index)), 'wb', buffering=0)
+            fd = open(join(self.dfuse.dir, 'file.{}'.format(index)), 'wb', buffering=0)
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations